nym-outfox/src/lib.rs

```diff
@@ -1,4 +1,10 @@
+use std::iter::repeat_with;
+
 pub mod error;
 pub mod format;
 pub mod lion;
 pub mod packet;
+
+pub fn randombytes(n: usize) -> Vec<u8> {
```

Comment:
imo if we're using random bytes, especially in crypto code, we should be accepting a generic `R: Rng + CryptoRng`